static/app/views/alerts/issueRuleEditor/ticketRuleModal.tsx

```diff
@@ -173,6 +174,28 @@ class TicketRuleModal extends AbstractExternalIssueForm<Props, State> {
     );
   };
 
+  getErrors() {
+    const errors: {[key: string]: React.ReactNode} = {};
+    for (const field of this.cleanFields()) {
+      if (field.type === 'select' && field.default) {
+        let found;
+        if (Array.isArray(field.default)) {
+          found = (field.choices || []).find(([value, _]) =>
+            ((field.default as Choices[0]) || []).includes(value)
+          );
+        } else {
+          found = (field.choices || []).find(([value, _]) => value === field.default);
+        }
```

Comment:
I believe this can be shortened to something like:
```ts
const fieldChoices = (field.choices  || []) as Choices;
const found = fieldChoices.find(([value, _]) =>
  Array.isArray(field.default)
    ? field.default.includes(value)
    : value === field.default
  );
```
but you might wanna double check my code.